### PR TITLE
[GSoC] Add reference implementation for Reviews_Declined

### PR DESCRIPTION
**Pull request by Polaris000**
This pull request adds the Reviews_Declined reference implementation.

### CODE REVIEW

**Comment by Polaris000** [comment]
@jgbarah , though I have not added the test, if possible, please review the python script.
Thanks!

File: implementations/code_df/reviews_declined_github.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    def compute(self):
+        """
+        Compute the total number of reviews which were declined, from the
+        Perceval data. Declined reviews are in the "closed" state but
+        are not merged.
+
+        :returns count: The total number of reviews declined
+        """
+
+        count = len(self.df['hash'][~self.df['merged']
```
Comment (jgbarah) [inline implementations/code_df/reviews_declined_github.py]: Why do you need `unique`? Sorry if it is obvious, but I don't get it...

Comment (Polaris000) [reply implementations/code_df/reviews_declined_github.py]: I don't think its required. I'm trying to think of a case when there would be multiple prs with the same id number, similar to how commits are the same in different forks.

Comment (Polaris000) [reply implementations/code_df/reviews_declined_github.py]: I'm removing it for now.

**Comment by jgbarah** [comment]
I think we're only missing tests now, right?

**Comment by Polaris000** [comment]
Yes, @jgbarah. WIll add those according to the latest changes to support travis.

**Comment by Polaris000** [comment]
@jgbarah, added tests. Please have a look. This pr will successfully pass travis, once #222 is merged.

**Comment by Polaris000** [comment]
This will pass only if #222 passes.

**Comment by jgbarah** [comment]
> This will pass only if #222 is approved.

You're right. But i just merged that one...

**Comment by Polaris000** [comment]
@jgbarah , sorry for the merge conflict. I fixed it. Please have a look. Travis passed.

**Comment by jgbarah** [comment]
> @jgbarah , sorry for the merge conflict. I fixed it. Please have a look. Travis passed.

Great! Merging...

**Pull request merged**